tests/layer_tests/conftest.py

```diff
@@ -89,6 +94,12 @@ def use_new_frontend(request):
     return request.config.getoption('use_new_frontend')
 
 
+@pytest.fixture(scope="session")
+def use_legacy_frontend(request):
+    """Fixture function for command-line option."""
+    return request.config.getoption('use_legacy_frontend')
```

Comment:
How it will go through tests?
Now I see that "if":
https://github.com/openvinotoolkit/openvino/blob/master/tests/layer_tests/common/layer_test_class.py#L57

It says that we will use legacy fe only in case of use_new_frontend=False

Reply:
in configs we determine which option will be used - use_new_frontend or use_legacy_frontend